library/src/com/squareup/timessquare/MonthView.java

```diff
@@ -57,6 +60,21 @@ public static MonthView create(ViewGroup parent, LayoutInflater inflater,
     return view;
   }
 
+  private static int getDayOfWeek(int firstDayOfWeek, int offset, boolean isRtl) {
+    int dayOfWeek = firstDayOfWeek + offset;
+    if (isRtl) {
+      return 8 - dayOfWeek;
+    }
+    return dayOfWeek;
+  }
+
+  private static boolean isRTL(Locale locale) {
```

Comment:
`isRtl`, acronyms always get camel cased to eliminate ambiguity: https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s5.3-camel-case